regression/test.pl

```diff
@@ -324,9 +339,13 @@ ($$$$)
 use Getopt::Long qw(:config pass_through bundling);
 $main::VERSION = 0.1;
 $Getopt::Std::STANDARD_HELP_VERSION = 1;
-our ($opt_c, $opt_e, $opt_f, $opt_i, $opt_j, $opt_n, $opt_p, $opt_h, $opt_C, $opt_T, $opt_F, $opt_K, $opt_s, %defines, @include_tags, @exclude_tags); # the variables for getopt
-GetOptions("D=s" => \%defines, "X=s" => \@exclude_tags, "I=s" => \@include_tags);
-getopts('c:efi:j:nphCTFKs:') or &main::HELP_MESSAGE(\*STDOUT, "", $main::VERSION, "");
+our ($opt_c, $opt_e, $opt_f, $opt_i, $opt_j, $opt_n, $opt_p, $opt_h, $opt_C, $opt_T, $opt_F, $opt_K, $opt_s, $opt_S, %defines, @include_tags, @exclude_tags); # the variables for getopt
+$color_output_enabled = 1;
+if (exists $ENV{'TESTPL_COLOR_OUTPUT'}) {
+  $color_output_enabled = $ENV{'TESTPL_COLOR_OUTPUT'};
+}
+GetOptions("D=s" => \%defines, "X=s" => \@exclude_tags, "I=s" => \@include_tags, 'color!' => \$color_output_enabled);
```

Comment:
Is it worth adding a comment to explain that the test/setting of value from the ENV *must* come before the call to GetOptions to preserve the behaviour that the command line option overrides the ENV variable?

Reply:
done